src/components/AddComment.vue

```diff
@@ -54,19 +55,27 @@ export default {
     async addComment() {
       try {
         this.loading = true;
-        await createComment({
+        const res = await createComment({
           blog_id: this.blogId,
           body: this.userComment,
         });
-        this.userComment = '';
-        this.$refs.observer.reset();
-        this.$notify({
-          title: this.$t('general.notify.succesTitle'),
-          message: this.$t('addComment.notify.succesMessage'),
-          type: 'success',
-        });
 
-        await this.$parent.getComments();
+        if (has(res, 'message'))
```

Comment:
Is it necessary to use has methond from lodash, can't you just use the respone from the endpoint